static/bidder-params/criteo.json

```diff
@@ -23,6 +23,16 @@
             "type": "integer",
             "description": "Impression's network ID, preferred.",
             "minimum": 0
+        },
+        "pubid": {
+            "type": "string",
+            "description": "Impression's publisher ID.",
+            "minLength": 1
+        },
+        "uid": {
+            "type": "integer",
+            "description": "Impression's ad unit id.",
+            "minimum": 0

```

Comment:
This will make more sense if you keep it in sync with `ExtImpCriteo`, even if you don't use it in code.

Reply:
Agreed, please update `openrtb_ext/imp_criteo.go` so that all fields listed in this file appear in `ExtImpCriteo`.

Reply:
Thanks for the review, I have updated the pull request.